tests/test_adjoint_jacobian.py

```diff
@@ -250,18 +252,29 @@ def test_multiple_rx_gradient(self, tol):
     ops = {qml.RX, qml.RY, qml.RZ, qml.PhaseShift, qml.CRX, qml.CRY, qml.CRZ, qml.Rot}
 
     @pytest.mark.parametrize("obs", [qml.PauliX, qml.PauliY])
-    @pytest.mark.parametrize("op", ops)
+    @pytest.mark.parametrize(
+        "op",
+        [
+            qml.RX(0.4, wires=0),
+            qml.RY(0.6, wires=0),
+            qml.RZ(0.8, wires=0),
+            qml.CRX(1.0, wires=[0, 1]),
+            qml.CRY(2.0, wires=[0, 1]),
+            qml.CRZ(3.0, wires=[0, 1]),
+            qml.Rot(0.2, -0.1, 0.2, wires=0),
+        ],
+    )
     def test_gradients(self, op, obs, tol, dev):
         """Tests that the gradients of circuits match between the finite difference and device
         methods."""
-        args = np.linspace(0.2, 0.5, op.num_params)
 
+        # op.num_wires and op.num_params must be initialized a priori
         with qml.tape.JacobianTape() as tape:
             qml.Hadamard(wires=0)
             qml.RX(0.543, wires=0)
             qml.CNOT(wires=[0, 1])
 
-            op(*args, wires=range(op.num_wires))
+            op
```

Comment:
Just curious if the operation is being applied here with this? Or is this a GH formatting issue that I cannot see the rest of the changes?

Reply:
Previously, `op` was defined as a list of operations' inits and this test function worked properly as values of `op.num_params` and `op.num_wires` were known for each operation. Now, with the recent updates, these values are not known and one must pass arguments a priori. Hence, each operation is being applied here with each `op` call without any needs on passing args and wires.